pkg/api/frontendsettings.go

```diff
@@ -329,6 +329,9 @@ func (hs *HTTPServer) getFrontendSettings(c *contextmodel.ReqContext) (*dtos.Fro
 	if hs.Cfg.UnifiedAlerting.Enabled != nil {
 		frontendSettings.UnifiedAlertingEnabled = *hs.Cfg.UnifiedAlerting.Enabled
 	}
+	if hs.Cfg.UnifiedAlerting.RecordingRules.Enabled {
+		frontendSettings.UnifiedAlerting.GrafanaRecordingRulesUrl = hs.Cfg.UnifiedAlerting.RecordingRules.URL
```

Comment:
I wonder if we should expose the full URL to the frontend. I guess it might contain sensitive information like username and password? 
Perhaps, we could expose matching data source UID instead?

Reply:
I would say that the username and password are in a separate field https://github.com/grafana/grafana/blob/ec89854c5e541adf87d110845295ebf4784bec01/pkg/setting/setting_unified_alerting.go#L112